packages/ListBox/src/components/Popover/Popover.js

```diff
@@ -21,7 +21,13 @@ export default function Popover(props) {
   const { children, zIndex } = props;
 
   return (
-    <sc.PopoverWrapper {...props} isOpen={state.isOpen} maxWidth={state.triggerWidth} offset={0} zIndex={zIndex}>
+    <sc.PopoverWrapper
+      {...props}
+      isOpen={Boolean(state.isOpen)}
```

Comment:
why not set the default value to false? I didn't find anywhere is using the `null` value for `isOpen`

Reply:
I think that even if we change it to false, that assumed we understand that `isOpen` can receive any falsy value which is not true, Adding Boolean protect us from falling into the trap of the Popover, I think we should fix the Popover and not use null to detect if the component is a control or uncontrolled. 

So is also protecting us from a silent and difficult bug to catch.

Reply:
I changed the value to false